provider/lib/details_screen.dart

```diff
@@ -0,0 +1,101 @@
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+import 'package:provider_sample/todo_list_model.dart';
+import 'package:todos_app_core/todos_app_core.dart';
+
+import 'edit_todo_screen.dart';
+import 'models.dart';
+
+class DetailsScreen extends StatelessWidget {
+  final String id;
+  final void Function() onRemove;
+
+  const DetailsScreen({@required this.id, @required this.onRemove})
+      : super(key: ArchSampleKeys.todoDetailsScreen);
+
+  @override
+  Widget build(BuildContext context) {
+    final model = Provider.of<TodoListModel>(context, listen: true);
```

Comment:
`listen` is `true` by default

```suggestion
    final model = Provider.of<TodoListModel>(context);
```